src/full_node/full_node.py

```diff
@@ -860,6 +866,81 @@ async def reject_compact_proof_of_time_request(
         for _ in []:
             yield _
 
+    async def broadcast_uncompact_blocks(
+        self, uncompact_interval, delivery: Delivery = Delivery.BROADCAST
+    ):
+        while self.sync_store.get_sync_mode():
```

Comment:
It it possible that we enter the sync mode later, so this check should be done within the loop below.